### ISSUE TITLE
Free Orbit with embedded viewer engine

### ISSUE THREAD

**Issue by fedemengo** [kovacsv/Online3DViewer#322]
This tool is awesome, huge kudos.

I've followed these [steps](https://github.com/kovacsv/Online3DViewer/wiki/Embed-the-Viewer-Engine#how-to-embed-the-engine) to include the engine in my blog. I'd like to configure it to use Free Orbit but I'm having problem figuring out how to that with my set up. 

Do you have any pointers? It doesn't seem to be configurable inside the `div.online_3d_viewer`.
Thank you in advance

**Comment by kovacsv** [comment]
Try following the "Embed with coding" section, and then call this function on the viewer:
```js
viewer.SetFixUpVector (false);
```

**Comment by fedemengo** [comment]
Did something change in the API? If I copy paste the snippet [here](https://github.com/kovacsv/Online3DViewer/wiki/Embed-the-Viewer-Engine#embed-with-coding) I get `Uncaught TypeError: OV.Color is not a constructor`.

I tried replacing `new OV.Color` with `new OV.RGBColor` but the model doesn't show up. The `onModelLoaded` callback fires but I don't see anything

**Comment by fedemengo** [comment]
Yep, I missed `RGBAColor` specifically for the background, I also had to omit the definition of the camera (I guess it wasn't correctly aligned with my image and omitting it let the library use a reasonable default value). 

Finally, I was able to use free orbit with `viewer.viewer.SetFixUpVector (false);` instead of `viewer.SetFixUpVector (false);` where `let viewer = new OV.EmbeddedViewer(parentDiv, ... )`.

Thank you very much for the help! Again, awesome work!
